dags/burnham.py

```diff
@@ -33,7 +33,8 @@
   FROM
     `{project_id}.burnham_live.{table}`
   WHERE
-    submission_timestamp > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 3 HOUR)
+    submission_timestamp BETWEEN TIMESTAMP_SUB(@burnham_start_timestamp, INTERVAL 1 HOUR)
```

Comment:
This should work as written if we pass the param as a TIMESTAMP as discussed in https://github.com/mozilla/burnham/pull/74/files#r496722221 otherwise I think we'll need to wrap all uses in `TIMESTAMP()`

Reply:
I changed the type in burnham-bigquery! 👍